envy/lib/config/envy_config.py

```diff
@@ -52,5 +52,11 @@ def get_setup_steps(self) -> [{}]:
     def get_actions(self) -> [{}]:
         return self.data["actions"]
 
+    def get_network_mode(self) -> Optional[str]:
+        return "host" if self.data["network"] else None
```

Comment:
I think I'm reading this wrong.

Doesn't this ignore `self.data["network"]`? Why is it returning "host" if there's a value in there?

Reply:
oh ya good catch! I think you're right